db/seeds.rb

```diff
@@ -5,23 +5,3 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-
-5.times do |x|
-  User.create!(
-    email: "user#{x}@example.com", 
-    password: "123456",
-    password_confirmation: "123456", 
-    role: ["admin", "superadmin"].sample,
-    phone_number:"06#{x}12#{x}3#{x}5#{x}"
-  )
-end
-
-25.times do |x|
-  Article.create!(
-    title: "Title #{x}",
-    user_id: rand(User.first.id..User.last.id),
-    text: "Text #{x} Words go here Idk",
-    language: ["ukr", "eng"].sample,
-    status: ["draft", "published", "trashed"].sample
-  )
-end
```

Comment:
why you removed it?

Reply:
I'm going to learn about Factory_bot to know how to use factories.

Reply:
At this moment I used only Faker. How to create factories and use them in tests - I still learning about this.